build.assets/Dockerfile

```
@@ -44,7 +44,7 @@ RUN git clone --depth=1 https://github.com/openssl/openssl.git -b openssl-3.0.8
     cd openssl && \
     [ "$(git rev-parse HEAD)" = '31157bc0b46e04227b8468d3e6915e4d0332777c' ] && \
     ./config --release --libdir=/usr/local/lib && \
-    make && \
+    make -j$(grep -c processor /proc/cpuinfo) && \
```

nit: Quote properly.

```suggestion
    make -j"$(grep -c processor /proc/cpuinfo)" && \
```

We could also do `make -j"$(nproc)"` (equivalent but a bit simpler), or just pick a number and go with it (like `make -j8`).

---

Simplified with `nproc`. I don't want to "hardcode" any number as these days even our CI builders have the range between 8-32.